README.md

```diff
@@ -107,7 +132,7 @@ There is a handy script that will build and push an image to
 and tag the source:
 
 ```
-./hack/release.sh
+./hack/release_operator_sdk.sh
```

Comment:
can you document when we would be using /hack/release.sh and /hack/release_operator_sdk.sh

is release_operator_sdk for convenience for developers are is there a plan to actually use it for official releases?

Reply:
I don't think there's any compelling reason to keep `release_operator_sdk.sh`

Reply:
We do not need to bother release.sh, since it is the script called by prow job to release automatically. It is enough for users to use `ko publish` to build and push the image anywhere they need. They do not need to care the release process, and hask/release.sh is not designated to run manually.